README.md

```diff
@@ -0,0 +1,92 @@
+# Infrastructure-as-Code Compliance Management Framework (IACMF) Core
+
+## Existing Plugins
+
+### Model Creation Plugins
+
+### Model Refinement Plugins
+
+### Compliance Checking Plugins
+
+### Compliance Issue Fixing Plugins
+
+#### Bash-based Ubuntu Issue Fixing Plugin
+
+**Summary**: This plugin allows executing bash commands on an ubuntu (virtual-)machine via ssh for the purpose of fixing
+a detected compliance issue.
+
+**Plugin Identifier:** `bash-fixing-plugin`
+
+**Possible Inputs for the Plugin**
+
+1. _From the Issue properties_
+
+   The plugin expects the `"CHECKER_COMPONENT_ID"` property. This property points to a component of type `Compute` in
+   the instance model that has the issue.
+   The referred component is expected to have the following properties:
+    - `os_family`: with the value `linux`.
+    - `machine_image`: with the value `ubuntu`.
+    - `public_address`: with the hostname/ip address of the ubuntu (virtual-)machine.
+    - (optional) `private_key_path`: with the path (on the server) to the private key that allows connecting to the (
+      virtual-)machine via ssh. If this is property has a value, the plugin configuration
+      entry `default-private-key-path` (see below) is ignored.
+
+
+2. _From the Configuration Entries_
+
+   The plugin has the following configuration entries:
+    - `script`: the bash script to be executed via ssh.
```

Comment:
will this be a file reference later? the complete file as text?

Reply:
only the command to be executed using the `VirtualMachine.execCommand` method.